js/plugins/oe_oembed_entities/plugin.js

```diff
@@ -94,6 +130,33 @@
           });
         }
       }
+
+      // Execute widget editing action on double click.
+      editor.on('doubleclick', function (evt) {
+        var element = getSelectedEmbeddedEntity(editor) || evt.data.element;
+
+        if (isEditableEntityWidget(editor, element)) {
+          editor.execCommand('edit_entities');
+        }
+      });
+
+      if (editor.contextMenu) {
+        editor.addMenuGroup('oembed');
+        editor.addMenuItem('edit_display', {
+          label: Drupal.t('Edit display options'),
+          command: 'edit_entities',
+          group: 'oembed'
+        });
+
+        editor.contextMenu.addListener(function (element) {
+          if (element && element.$ && element.$.firstChild) {
+            var child = element.$.firstChild;
+            if (child.hasAttribute('data-oembed')) {
+              return { edit_display: CKEDITOR.TRISTATE_OFF };
+            }
+          }
+        });
```

Comment:
Theoretically better to recheck that the menu and contextmenu plugins are loaded as it is done in core/modules/ckeditor/js/plugins/drupallink/plugin.es6.js. You can check the comment bellow
```suggestion
      if (editor.addMenuItem) {
        editor.addMenuGroup('oembed');
        editor.addMenuItem('edit_display', {
          label: Drupal.t('Edit display options'),
          command: 'edit_entities',
          group: 'oembed'
        });
      }
      if (editor.contextMenu) {
        editor.contextMenu.addListener(function (element) {
          if (element && element.$ && element.$.firstChild) {
            var child = element.$.firstChild;
            if (child.hasAttribute('data-oembed')) {
              return { edit_display: CKEDITOR.TRISTATE_OFF };
            }
          }
        });
```